app/src/main/java/com/github/codetanzania/ui/fragment/IssueDetailsFragment.java

```diff
@@ -1,110 +1,116 @@
-        statusesRecyclerView.setAdapter(issueProgressTimelineAdapter);
-        LinearLayoutManager manager = new LinearLayoutManager(getActivity(), LinearLayoutManager.VERTICAL, false);
-        statusesRecyclerView.setLayoutManager(manager);
-    }
-}
+package com.github.codetanzania.ui.fragment;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.github.codetanzania.Constants;
+import com.github.codetanzania.adapter.IssueItemsViewPagerAdapter;
+import com.github.codetanzania.adapter.IssueProgressTimelineAdapter;
+import com.github.codetanzania.model.Comment;
+import com.github.codetanzania.model.ServiceRequest;
+import com.github.codetanzania.util.Util;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+import me.relex.circleindicator.CircleIndicator;
+import tz.co.codetanzania.R;
+
+public class IssueDetailsFragment extends Fragment {
+
+    private static final String TAG = "IssueDetailsFragment";
+
+    // reference to the recycler view. used to show map and image captured when
+    // user submitted an issue
+    // private RecyclerView mAttachmentsRecyclerView;
+
+    @Override
+    public View onCreateView(
+            LayoutInflater inflater, ViewGroup group, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.frag_issue_details, group, false);
+    }
+
+    public static IssueDetailsFragment getInstance(Bundle args) {
+        IssueDetailsFragment inst = new IssueDetailsFragment();
+        inst.setArguments(args);
+        return inst;
+    }
+
+    @Override
+    public void onViewCreated(View fragView, Bundle savedInstanceState) {
+
+        Bundle args = getArguments();
+        ServiceRequest serviceRequest = args.getParcelable(Constants.Const.TICKET);
+        assert serviceRequest != null;
+
+        int numFrags = serviceRequest.attachments == null || serviceRequest.attachments.isEmpty() ? 1 : 2;
+
+        // debug
+        Log.d(TAG, String.format("%d", numFrags));
+
+        // view pager
+        ViewPager viewPager = (ViewPager) fragView.findViewById(R.id.viewPager);
+        IssueItemsViewPagerAdapter viewPagerAdapter = new IssueItemsViewPagerAdapter(
+                getChildFragmentManager(), serviceRequest, numFrags);
+        viewPager.setAdapter(viewPagerAdapter);
+        CircleIndicator indicator = (CircleIndicator) fragView.findViewById(R.id.indicator);
+        indicator.setViewPager(viewPager);
+
+        // bind description data
+        TextView tvIssueDate = (TextView) fragView.findViewById(R.id.tv_IssueDate);
+        tvIssueDate.setText(Util.formatDate(serviceRequest.createdAt, "yyyy-MM-dd HH:mm:ss"));
```

Comment:
Ideally: this util method should include the simple date format, so that it can be consistent throughout the app.